copy-assets.sh

```diff
@@ -0,0 +1 @@
+cp -r node_modules/@cursorless/tree-sitter-wasms/out parsers
```

Comment:
Tempted to just include `node_modules/@cursorless/tree-sitter-wasms/out` directory in the extension bundle and read them in place to avoid this step but I don't feel strongly

Reply:
I took a crack at this but it apparently isn't recommended and you do get an error

```
The following include patterns in the "files" property in package.json do not match any files packaged in the extension:
  - node_modules/@cursorless/tree-sitter-wasms/out
Remove any include pattern which is not needed.
```

I think copying assets is fine.